client/my-sites/media-library/list-plan-upgrade-nudge.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import UpgradeNudgeExpanded from 'blocks/upgrade-nudge-expanded';
+import { PLAN_PREMIUM, FEATURE_VIDEO_UPLOADS, FEATURE_AUDIO_UPLOADS } from 'lib/plans/constants';
+
+export const MediaLibraryUpgradeNudge = ( { translate, filter } ) => {
+	let title = translate( 'Upgrade to a Premium Plan and Enable VideoPress' );
+	let subtitle = translate( `By upgrading to a Premium Plan you'll enable VideoPress support on your site.` );
+	let highlightedFeature = FEATURE_VIDEO_UPLOADS;
+	let benefits = [
+		translate(
+			'Upload videos to your site with an interface designed specifically for WordPress.',
+			{ comment: 'Perk for upgrading to premium plan' }
+		),
+		translate(
+			'Present videos using a lightweight and responsive player that is ad-free and unbranded.',
+			{ comment: 'Perk for upgrading to premium plan' }
+		),
+		translate(
+			'See where your videos have been shared as well as stats for individual and overall video plays.',
+			{ comment: 'Perk for upgrading to premium plan' }
+		)
+	];
+
+	if ( 'audio' === filter ) {
```

Comment:
The length of this stateless function make it difficult to read. Can you move some of this to helper functions?